roles/dbserver/tasks/main.yml

```diff
@@ -35,3 +35,8 @@
 - import_tasks: fix_template_encoding.yml
   tags: template_encoding
 
+- name: install wal2json module
+  include_role:
+    name: libre_ops.wal2json
+  vars:
+    wal2json_set_preload_libararies: false
```

Comment:
:warning: isn't `libararies` a typo?

Reply:
That's how it's spelled in the role too. :see_no_evil: I'll fix it.

Reply:
Done: https://github.com/openfoodfoundation/ofn-install/pull/468/commits/2a16f81028f58590833423c1c0f35651977380cc